docs/guide/adapters/README.md

```diff
@@ -7,853 +7,17 @@ next: ../installation/
-    </details>
-
-
-* Serial connected
-
-  * <details>
-    <summary>Texas Instruments CC2530</summary>
-  
-    Serial connected adapter with external antenna optionally with CC2591 or CC2592 RF frontend  
-    **Warning 1:** requires additional hardware to flash (CC debugger + download cable)  
-    **Warning 2:** might not be powerful enough to handle networks of 20+ devices  
-    * [Coordinator firmware](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_Home_1.2/bin)  
-    * [Router firmware](https://github.com/Koenkk/Z-Stack-firmware/tree/master/router/Z-Stack_Home_1.2/bin)  
-    * [Flashing instructions](../../advanced/zigbee/05_create_a_cc2530_router.md#2-flashing-the-cc2530)  
-    * [Connecting](./flashing/connecting_cc2530.md)  
-    * Buy: [AliExpress](http://www.aliexpress.com/wholesale?catId=0&initiative_id=SB_20181213104041&SearchText=cc2530) [GBAN](http://www.gban.cn/en/product_show.asp?id=43) [Tindie](https://www.tindie.com/products/GiovanniCas/cc2530-cc2592-zigbee-dongle/)
-  
-    ![](../../images/cc2530.jpg)
-    </details>
-  * <details>
-    <summary>Texas Instruments CC2538</summary>
-  
-    Serial connected adapter with CC2592 RF Amplifier  
-    * [Coordinator firmware](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.0.x/bin)  
-    * [Flashing](./flashing/flashing_the_cc2538.md)  
-    * Buy: [AliExpress](https://www.aliexpress.com/wholesale?catId=0&initiative_id=SB_20191108075039&SearchText=cc2538)
-  
-    ![](../../images/cc2538.jpg)
-    </details>
-
-* Raspberry Pi
-
-  * <details>
-    <summary>Texas Instruments CC2538 HAT</summary>
-  
-    Raspberry pinout compatible HAT with CC2538 and optional external antenna  
-    * [Coordinator firmware](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.0.x/bin)  
-    * [Flashing](./flashing/flashing_the_cc2538.md)  
-    * [Buy](https://www.tindie.com/products/GiovanniCas/zigbee-hat-with-cc2538-for-raspberry/)
-  
-    ![](../../images/cc2538hat.jpg)
-    </details>
-  
-* Network
-
-  * <details>
-    <summary>XGG gateway</summary>
-  
-    An open source zigbee gateway powered by ESP8266 and CC2538+CC2592PA (XGG 38PZ2MGateway) or CC2530 (XGG 30Z2MGateway)  
-    Coordinator firmware: [XGG 38PZ2MGateway](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.0.x/bin) [XGG 30Z2MGateway](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_Home_1.2/bin)  
-    * [Contact](https://shop68536829.taobao.com/)
-  
-    ![](../../images/CC2652P-Z2M.jpg)
-    </details>
-
-
-## Experimental
-The adapters below are experimental, don't use these if you want a stable setup.
-
-### Based on Silicon Labs EFR32MG2x/MGM2xx and EFR32MG1x/MGM1xx series
-
-There is experimental (a.k.a. alpha stage) support for Silicon Labs based Zigbee adapters with Silabs EmberZNet NCP 6.7.8 firmware or newer via EZSP version 8 (EmberZNet Serial Protocol) interface or later.
```

Comment:
With this MR, the message about Silicon Labs chips having experimental support (in the README) disappeared. Is this true? Or is this a miss?

Many places (including zigbee coordinator manufacturers) still claim support is experimental (see https://smlight.tech/ )

![image](https://github.com/Koenkk/zigbee2mqtt.io/assets/134083/7b37b863-f4ce-44f8-b001-0b10d70c1b04)

Reply:
Its not experimental anymore